solidity/contracts/LPRewards.sol

```diff
@@ -270,3 +270,13 @@ contract LPRewardsTBTCv2Saddle is LPRewardsTBTCSaddle {
         LPRewardsTBTCSaddle(keepToken, tbtcV2SaddleLPToken)
     {}
 }
+
+/// @title KEEP rewards for the tBTC V2 Saddle V2 liquidity pool.
+/// @dev LP token: TODO: Fill
+///      Pool: TODO: Fill
```

Comment:
https://docs.saddle.finance/contracts#tbtcv2-metapool-v2

LP token is `0x3f2f811605bC6D701c3Ad6E501be13461c560320`
Pool is `0xA0b4a2667dD60d5CdD7EcFF1084F0CeB8dD84326`

Reply:
Let's also update pool address for v1 `LPRewardsTBTCv2Saddle` to `0xf74ebe6e5586275dc4CeD78F5DBEF31B1EfbE7a5` instead of a link to Saddle dApp.